README.md

```diff
@@ -25,18 +25,26 @@
 
 ## Overview
 
+neo-python is an alternative implementation of the original NEO VM and can be run un
+*nix and Windows environments.
```

Comment:
Currently it can't easily be run on Windows (because of LevelDB bindings not being easy to install on Windows).